src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    public static final String DEFAULT_COLOR = "White";
```

Comment:
Replace DEFAULT_COLOR and DEFAULT_RADIUS fo FigureSupplier

Also, add WHITE to Color enum and use it for constant DEFAULT_COLOR